weatherbenchX/metrics/wrappers.py

```diff
@@ -63,15 +63,15 @@ def binarize_thresholds(
   Returns:
     binary_x: Binarized DataArray.
   """
-  if not isinstance(thresholds, xr.DataArray):
+  if not isinstance(thresholds, xr.Dataset):
```

Comment:
In `binarize_thresholds` def, you can update the typing of thresholds to `Iterable[float] | xr.Dataset`